src/sections/create-collection/collection-form/metadata-fields-section/metadata-input-level-fields-block/input-levels-table/InputLevelsTable.tsx

```diff
@@ -26,7 +26,7 @@ export const InputLevelsTable = ({
       })}>
       <div className={styles['close-button-container']}>{closeButton}</div>
 
-      <Table bordered={false}>
+      <Table>
         <tbody>
```

Comment:
I removed it because of an error from tsc:
<img width="1144" alt="Screenshot 2024-09-03 at 9 36 22 AM" src="https://github.com/user-attachments/assets/3c196258-d02a-42d3-9ecb-5f416b345101">

Reply:
Yeah, but it is because you need to rebuild the design system, which will update the Table props interface, and if tsc error remains you can close and open your ide.

Reply:
ok, sorry about that, I should have asked you. It's working now.

Reply:
don't worry at all!
Closing and opening the IDE after rebuilding the design system was necessary in your case? or is it something that only happens with vscode maybe..

Reply:
With Webstorm it is kind of unpredictable, sometimes it recognizes changes immediately, sometimes I have to refresh the file or restart, this time refreshing worked :)